azure-quantum/tests/unit/test_qiskit.py

```diff
@@ -290,6 +290,7 @@ def test_qiskit_submit_ionq_5_qubit_superposition(self):
         workspace = self.create_workspace()
         provider = AzureQuantumProvider(workspace=workspace)
         self.assertIn("azure-quantum-qiskit", provider._workspace.user_agent)
+        # backend = provider.get_backend("ionq.simulator", input_data_format = "ionq.circuit.v1", output_data_format = "ionq.quantum-results.v1", gateset="qis")
```

Comment:
Are we commenting this out just temporarily? Otherwise, if this is not required anymore we can just remove it

Reply:
removing